SU-SOE/stanford_bean_types
# removed soe-default view mode and updated the default view mode

# READY FOR REVIEW

# Summary
- See: https://stanfordits.atlassian.net/browse/SOE-3329 and https://stanfordits.atlassian.net/browse/SOE-3328

This removes the soe-default view mode and updates the default view mode to replace the the functionality of the soe-default view mode.

# Needed By (Date)
- Sprint end

# Criticality
- Client is eagerly awaiting this.


# Steps to Test
- Switch to this branch:
`git checkout 7.x-2.x-dev-remove-soe-default`
- Check if feature is overridden
` drush fd stanford_bean_types`
- And revert feature if necessary
`drush fr stanford_bean_types -y --force`
- Navigate to `admin/structure/block-types/manage/stanford-testimonial-block/display`
- Verify you see the `Default` view mode only and it looks like this:

![screen shot 2018-08-09 at 09 10 08](https://user-images.githubusercontent.com/284440/43911325-13fbb650-9bb4-11e8-8a0f-bec4d7bafcae.png)


# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3329
https://stanfordits.atlassian.net/browse/SOE-3328
PR ticket: https://stanfordits.atlassian.net/browse/SOE-3330

## Related PRs

## More Information

## Folks to notify
@annawatt
@kerri_augenstein


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
